addons/medical/functions/fnc_addPain.sqf

```diff
@@ -0,0 +1,28 @@
+/*
+ * Author: commy2
+ * Adds pain to unit and plays a scream sound.
```

Comment:
Actually removing pain is scaled down by the already present pain as well. I'll fix that too.